### PR TITLE
Adding GearID Geo Logo and entry into WhosUsingSvelte.js

### PR DESCRIPTION
**Pull request by mkamaalr**
Request to add GearID Geo logo to the list of "Who's Using Svelte".

Thank you!

### CODE REVIEW

**Comment by swyxio** [comment]
Deploy Preview is done!

Built with commit cddda46d2fee954fb6fd8fdd3611d052c76b69b4

https://deploy-preview-418--svelte-community.netlify.app

**Review by dreitzner — CHANGES_REQUESTED** [review]
- [ ] Please add trailing comma at the end of the list
- [ ] please add your entry add the end of the file
- [ ] please add your logo at max height of 72px or as svg

**Review by mkamaalr — COMMENTED** [review]
Deployed

File: whos-using-svelte/WhosUsingSvelte.js [hunk 1]
```diff
@@ -894,9 +894,9 @@ export const companies = [
 		href: "https://www.enablerr.ch/",
 		src: "organisations/enablerr.png",
 		alt: "enablerr by pier4all"
-	},
-	{
-		href: "https://thalasseus.com/",
+	},	
+  	{
```
Comment (dreitzner) [inline whos-using-svelte/WhosUsingSvelte.js]: There should be no changes here....

**Review by dreitzner — CHANGES_REQUESTED** [review]

**Review by dreitzner — APPROVED** [review]
Lgtm

**Review by dreitzner — APPROVED** [review]

**Pull request merged**